Add Game Screen-Shot

ashishsiot opened the pull request.

ashishsiot committed:
game play screenshort

ashishsiot committed:
Add Game Screenshot 

Add Game Screenshot To Understand How The Actual Game Look Link On Readme.md File  And Add Live Demo  Link Associated With Image Whenever We Click On  On Image We Redirect To The Live Demo Link

straker commented on README.md:
```diff
@@ -2,6 +2,9 @@ Title: endless-runner-html5-game<br>
 Website: http://blog.sklambert.com/endless-runner-html5-game/<br>
 Play Game: https://tutsplus.github.io/kandi-runner/<br>
+
+[![CarDEalerShip](https://raw.githubusercontent.com/ashishsiot/endless-runner-html5-game/master/game.PNG)](https://tutsplus.github.io/kandi-runner/)
```
What's with the `CarDEalerShip` image alt? Also, could you use this repos as the image (and not yours) or just paste the image into the file (github will resolve it and host it themselves).

ashishsiot replied:
I Make One Note Of All Syntax Of .md (Markdown Language )  And Whenever I Want To Make Changes In Any Ones Repo I Will Just Simply Copy Past The Syntax And Make Changes But This Time I Will Forget To Change Header Name.

And The About Image Address Will Just Upload Image Into My Fork Repo And Use This Hosted Image Address.   

- I Am Really Sorry About That I Am Still Learning And Thank You For Showing My Mistakes I Hope You Will Help Me To Grow Up 

-Thank You

ashishsiot committed:
Update Header Name

Update Header Name Of Game Screen-Shoot  Code That  Use In Readme.md File 
-Thank You

ashishsiot commented on README.md:
```diff
@@ -2,6 +2,9 @@ Title: endless-runner-html5-game<br>
 Website: http://blog.sklambert.com/endless-runner-html5-game/<br>
 Play Game: https://tutsplus.github.io/kandi-runner/<br>
+
+[![endless-runner](https://raw.githubusercontent.com/ashishsiot/endless-runner-html5-game/master/game.PNG)](https://tutsplus.github.io/kandi-runner/)
```
With New Header Name

straker replied:
Thanks for changing the image alt. 

Due to security reasons the image URL must be from this repo. Otherwise I don't control the actual image and (not that I'm saying you would) you could easily change the image on your end without my knowledge and it would then show in this repository. 

Since you've uploaded the image with the pull request, we just need change the url to this repo and it should work after merge.

```suggestion
[![endless-runner](https://raw.githubusercontent.com/straker/endless-runner-html5-game/master/game.PNG)](https://tutsplus.github.io/kandi-runner/)
```

straker committed:
Update README.md
